pegjs/snowflake.pegjs

```diff
@@ -508,6 +508,7 @@ create_domain_stmt
     }
 create_table_stmt
   = a:KW_CREATE __
+    orre: (KW_OR __ KW_REPLACE __)?
```

Comment:
CREATE OR REPLACE

Reply:
Should it be `or:` for consistency? As here: https://github.com/taozhi8833998/node-sql-parser/blob/d59020b0509fbcabc200b2951165597e90ffa0d7/pegjs/mysql.pegjs#L735